### PR TITLE
Multi seqs display

### PR DESCRIPTION
**Pull request by MImranAsghar**
Update select dropdown UI from (https://github.com/ProjectMirador/mirador/tree/2949-multi-seq). Update it to work with the latest master; add some tests for multi-seq disp.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.53",
     "@researchgate/react-intersection-observer": "^1.0.0",
+    "canvas": "^2.6.1",
```
Comment (mejackreed) [inline package.json:38]: Can you say more about this dependency that is added?

Comment (MImranAsghar) [reply package.json]: Sorry was testing something, just removed it

File: __tests__/integration/mirador/blank.html [hunk 2]
```diff
@@ -13,7 +13,9 @@
     <script type="text/javascript">
      var miradorInstance = Mirador.viewer({
        id: 'mirador',
-       windows: [],
+       windows: [{
+         manifestId: 'https://www.e-codices.unifr.ch/metadata/iiif/gau-Fragment/manifest.json'
```
Comment (mejackreed) [inline __tests__/integration/mirador/blank.html:17]: Can we remove the added manifest here, if needed we can create a new page "multiplesequence.html" or something similar?

Comment (MImranAsghar) [reply __tests__/integration/mirador/blank.html]: Hey, just pushed

**Review by mejackreed — APPROVED** [review]
Thanks for pushing this work forward @MImranAsghar 👏

**Pull request merged**